cumulusci.yml

```diff
@@ -359,6 +372,15 @@ flows:
         steps:
             3:
                 task: ensure_record_type
+    
+    trial_org:
+        steps:
+            1:
+                flow: dependencies
+            2:
+                task: install_managed
+            3:
+                flow: config_trial
```

Comment:
I'm not clear on the use case for adding this flow. It's the same as `install_prod`, isn't it?

Reply:
Yeah, it has less utility after admitting defeat on being able to activate CRLPs in code. I'll remove it for now but hope to bring it back later.